### PR TITLE
MWP-873: Create migration script for certs from old kube to new kube

### PR DESCRIPTION
**Pull request by russellmacshane**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,8 +11,10 @@ Cert Manager Migration from BoltDB to PostgreSQL
...
 7. Run this project
-    1. Migrate Data `go run migration.go migrate` 
-    1. Print Bolt Data `go run migration.go print-bolt-data`
-    2. Create DB Tables `go run migration.go create-tables`
-    3. Drop DB Tables `go run migration.go drop-tables`
+    1. Migrate Data `./migration.go migrate` 
+    2. Print Bolt Data `./migration.go print-bolt-data`
+    3. Create DB Tables `./migration.go create-tables`
+    4. Drop DB Tables `./migration.go drop-tables`
+    5. Migrate Certs and Secrets `./migration.go kube-migrate`
```
Comment (ctriv) [inline README.md:18]: If I run `go build migration.go` I get a binary named `migration` not `migration.go`.  Even nicer, if you just run `go build` you get a binary named `cert-manager-migration`.

File: project/types.go [hunk 2]
```diff
@@ -25,18 +32,83 @@ type UserInfo struct {
...
+		User string `yaml:"user"`
+		Password string `yaml:"password"`
+		SslMode string `yaml:"ssl_mode"`
+	}
+
+	Kube struct {
+		SourceConfigFile string `yaml:"src_config_file"`
+		DestConfigFile string `yaml:"dest_config_file"`
+	}
+}
```
Comment (ctriv) [inline project/types.go:59]: We've used [viper](https://github.com/spf1) for configuration on a bunch of other go projects.  Probably outside the scope of this PR, but something to think about.

**Review by ctriv — APPROVED** [review]
Looks good.

File: README.md (hunk 1 above)
Comment (russellmacshane) [reply README.md:18]: ahh, I've got some typos here. I like making it `cert-manager-migration`

File: project/types.go (hunk 2 above)
Comment (russellmacshane) [reply project/types.go:59]: Cool, I'll look into that for another PR.

**Pull request merged**